srfi-doc/srfi/scribblings/srfi-5-doc-free.scrbl

```diff
@@ -0,0 +1,165 @@
+#lang scribble/doc
+@(require srfi/scribblings/util
+          scribble/manual
+          scribble/example
+          scriblib/render-cond
+          scribble/core
+          scribble/html-properties
+          (for-syntax scheme/base)
+          (for-label (except-in scheme/base let)
+                     srfi/5
+                     racket/stream))
+
+@title[#:tag "srfi-5" #:style 'unnumbered]{
+ SRFI 5: A compatible let form with signatures and rest arguments}
+@defmodule[srfi/5]
+
+@begin[
+ (define-syntax-rule (defrkt rkt-let)
+   (begin
+     (require (for-label racket/base))
+     (define rkt-let (racket let))))
+ (defrkt rkt-let)
+ (define reference-doc
+   '(lib "scribblings/reference/reference.scrbl"))
+ (define guide-doc
+   '(lib "scribblings/guide/guide.scrbl"))
+ (define srfi-nf-doc
+   '(lib "srfi/scribblings/srfi-nf.scrbl"))
+ ]
+
+Original specification:
+@seclink[#:indirect? #t #:doc srfi-nf-doc srfi-5-std-taglet]{SRFI 5}
+
+For @hyperlink[srfi-license-history-url]{historical
+ reasons}, the SRFI 5 specification document has a
+@seclink[#:indirect? #t #:doc srfi-nf-doc srfi-5-license-taglet]{
+ restrictive license} and is not included in the main Racket distribution.
+
+The implementation in @racketmodname[srfi/5] and this
+documentation are distributed under the same
+@seclink["top" #:doc '(lib "scribblings/main/license.scrbl")]{license}
+as Racket: only the original specification document is
```

Comment:
This attempt to link to the Scribble document of the Racket license, and the analogous one in: <https://github.com/racket/srfi/blob/5bd7b1825ee6ee9d5b22cfdd406f82233afbf42a/srfi-doc-nonfree/srfi/scribblings/srfi-nf.scrbl#L54>
don't seem to work: they send me to a failed redirect page. I don't understand why.

Reply:
Is it possible that it doesn't have a section named `"top"`?

Reply:
Good thought! I don't know _why_ it doesn't seem to have a section named `"top"`—I thought every Scribble document did—but I don't understand all of the magic of the special pages. I inspected it and found it does have a section named `"License"`. I've pushed a new commit, 3111418c7b5677cb016b9125056fd4c1e3402b90, using that.

There's more confusion, though. To start with what does work, putting this program:
```racket
#lang scribble/manual
@seclink["License" #:doc '(lib "scribblings/main/license.scrbl")]{click here}
```
in DrRacket and clicking the `Scribble HTML` button reliably produces a document containing, e.g.:
```html
<a href="file:///home/philip/Downloads/racket-2022-01-26/racket-8.4.0.4/doc/license/index.html" data-pltdoc="x">click here</a>
```
which works great!

However, the link still isn't working when the documentation is installed by `raco setup`, neither as an update not as part of a fresh installation in an isolated environment. I've tried `raco cross`, snapshot installers, and Guix, with at least versions 8.3.900 and 8.4.0.4, and from Firefox, Falkon, and Konqueror. In each case, the link still tries to send me to a `local-redirect/index.html?doc=license&rel=index.html&version=8.4.0.4` (with various version numbers), either locally, on a snapshot site, or on `docs.racket-lang.org`, depending on how I've built Racket. In any case, the redirection does not work.